src/main/java/org/prebid/server/auction/requestfactory/Ortb2RequestFactory.java

```diff
@@ -517,10 +517,14 @@ private UpdateResult<String> resolveCountry(Geo geo, PrivacyContext privacyConte
     }
 
     private static UpdateResult<String> resolveRegion(Geo geo, PrivacyContext privacyContext) {
-        final String regionInRequest = geo != null ? geo.getRegion() : null;
+        final String regionInRequest = geo != null
```

Comment:
Suggestion.
What if we will a bit refactor this method
``` java
final String regionInRequest = Optional.ofNullable(geo)
                        .map(Geo::getRegion)
                        .map(String::toUpperCase)
                        .orElse(null);
```

Same can be done for geoInfo

Reply:
Let leave it as it is now, because it's aligned with the other class code style